tests/runtime/other

```diff
@@ -174,3 +174,8 @@ NAME map_assign_map_ptr
 RUN bpftrace -e 'i:ms:100 { @ = curtask; @a = @; printf("%d\n", @a); exit(); }'
 EXPECT -?[0-9]+
 TIMEOUT 1
+
+NAME runtime_error_check
+RUN bpftrace -kk -e 'i:ms:100 { @[1] = 1; printf("%d\n", @[2]); exit(); }'
+EXPECT 0
```

Comment:
Wouldn't it make more sense to look for the warning message?

Reply:
Yes, I updated it. (I didn't realize that the runtime engine redirected stderr to stdout and simply did regex match using EXPECT.)